libsplinter/src/admin/service/mod.rs

```diff
@@ -797,7 +797,8 @@ mod tests {
             .start()
             .expect("Unable to start Connection Manager");
         let connector = cm.connector();
-        let mut peer_manager = PeerManager::new(connector, None, Some(1), "test-node".to_string());
+        let mut peer_manager =
```

Comment:
All the test should fail if ref counting is violated, so I think these should all be true?

Reply:
whoops, yep you are correct